nekoyume/Assets/_Scripts/BlockChain/RPCAgent.cs

```diff
@@ -209,10 +209,6 @@ private IEnumerator CoJoin(Action<bool> callback)
             }
 
             // 상점의 상태를 한 번 동기화 한다.
-            States.Instance.SetShopState(
```

Comment:
Is it okay that we remove it?

Reply:
> Is it okay that we remove it?

It calls `SetShopState()` every time you enter the shop, so it doesn't seem necessary here. :thinking_face: